src/Servers.jl

```diff
@@ -421,7 +421,11 @@ function handle_transaction(f, t::Transaction, server; final_transaction::Bool=f
         @debug 2 "server closewrite"
         closewrite(http)
     catch e
-        @error "error handling request" exception=(e, stacktrace(catch_backtrace()))
+        # The remote can close the stream whenever it wants to, but there's nothing
+        # anyone can do about it on this side. No reason to log an error in that case.
+        level = e isa Base.IOError && !isopen(http) ? Logging.Debug : Logging.Error
```

Comment:
Since only the loglevels are needed you can directly use (and remove the Logging dep):
```suggestion
        level = e isa Base.IOError && !isopen(http) ? Base.CoreLogging.Debug : Base.CoreLogging.Error
```

Reply:
Yes, but isn't that technically internal? Unlikely to break, I suppose, but still...

Reply:
I guess, although `@logmsg` is from Base so :shrug: Anyway, a stdlib dependency costs nothing.